db/migrate/20230214104851_remove_total_amount_from_categories.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveTotalAmountFromCategories < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :categories, :TotalAmount, :decimal
+  end
+end
```

Comment:
- [ ] When I was running the migrations, this specific migration was causing the migrations to stop. The reason when I inspected was that I realized that in the table categories, there was no column such as "TotalAmount" that was created when creating that table. So having this migration that deletes that column brings confusion to Rails. It was resolved by simply deleting the migration as it as it is redundant.